src/include/rbd/features.h

```diff
@@ -12,6 +12,7 @@
 #define RBD_FEATURE_OPERATIONS          (1ULL<<8)
 #define RBD_FEATURE_MIGRATING           (1ULL<<9)
 #define RBD_FEATURE_NON_PRIMARY         (1ULL<<10)
+#define RBD_FEATURE_IMAGE_CACHE         (1ULL<<11)
```

Comment:
It was _IMAGE_ because it's the ImageCache interface (not the block dispatch interface), but now that you mention it users probably don't know or care what interface this uses.

RBD_FEATURE_CLIENT_CACHE?

Reply:
How about RBD_FEATURE_DIRTY_CACHE?

Reply:
Either works for me -- DIRTY_CACHE better implies that it's a transient state perhaps.

Reply:
Updated it to RBD_FEATURE_DIRTY_CACHE.